addons/calico-vxlan/calico-vxlan.yaml

```diff
@@ -4316,6 +4446,9 @@ spec:
             # Enable or Disable VXLAN on the default IP pool.
             - name: CALICO_IPV4POOL_VXLAN
               value: "CrossSubnet"
+            # Enable or Disable VXLAN on the default IPv6 IP pool.
+            - name: CALICO_IPV6POOL_VXLAN
+              value: "CrossSubnet"
```

Comment:
Since we don't support dual-stack or IPv6, should we disable this option?

Reply:
I think that in case that IPv6 is not enabled (there are no IPv6 pools) this should not be important (should have no effect). Since `CALICO_IPV4POOL_VXLAN` is set to `CrossSubnet` as well, I think it is dafe to just keep it like it is.